conda/recipes/blazingsql/meta.yaml

```diff
@@ -20,54 +20,39 @@ source:
 
 requirements:
     build:
-        - gtest
-        - gmock
-        - google-cloud-cpp 1.16.0
         - cmake
-        - cppzmq
-        - python
-        - pip
-        - setuptools
-        - spdlog >=1.7.0,<1.8.0
-        - cython >=0.29,<0.30
-        - openjdk >=8.0
-        - maven
         - ninja
-        - dask-cudf {{ minor_version }}.*
-        - dask-cuda {{ minor_version }}.*
-        - cudatoolkit {{ cuda_version }}.*
-    host:
         - gtest
-        - gmock
-        - google-cloud-cpp 1.16.0
-        - cmake
+        - aws-sdk-cpp
+        - google-cloud-cpp 1.16.0.*
+        #- nlohmann_json
```

Comment:
This was from trying to upgrade to 1.18.0 which has a bug of not including this automatically, see: https://github.com/conda-forge/google-cloud-cpp-feedstock/issues/28